frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -422,11 +502,82 @@ export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps
   );
   const NoDataEmptyMsg = () => <MsgBox title="No Operators Found" detail={noDataDetail} />;
 
-  return (
+  const isGlobalOperator = (source, kind) => {
+    return (
+      referenceForModel(ClusterServiceVersionModel) === kind && source.status?.reason === 'Copied'
+    );
+  };
+
+  const removedDuplicateGlobalOperators = (operators: any[]) => {
+    return operators.length > 0
+      ? operators.filter((source) => {
+          const kind = referenceFor(source);
+          if (source == null) {
+            return false;
+          }
+          if (isSubscription(kind)) {
+            return true;
+          }
+          if (isGlobalOperator(source, kind)) {
+            return false;
+          }
+          return true;
+        })
+      : [];
+  };
+
+  const filteredData = removedDuplicateGlobalOperators(data);
+
+  const managedNamespaceSort = (obj: ClusterServiceVersionKind | SubscriptionKind): string => {
+    const namespaces = obj.metadata?.annotations?.['olm.targetNamespaces'];
+    const namespacesList = namespaces == null ? [] : namespaces.split(',');
+    let olmManagedNamespace;
+    if (namespacesList.length === 0) {
+      olmManagedNamespace = 'All Namespaces';
+    } else {
+      olmManagedNamespace =
+        namespacesList.length > 1 ? `${namespacesList.length} Namespaces` : namespacesList;
+    }
+    if (obj.kind === 'Subscription') {
+      olmManagedNamespace = 'None';
+    }
```

Comment:
@rachael-phillips and @tlwu2013 suggested using 'None' for the Subscriptions row data for Managed Namespace. This is keeping it consistent to that value. I can refactor the function though.

Reply:
I guess I'm thinking about the case where `None` will flicker and then be replaced when the operator is first installed (since the Subscription will exist before the CSV).

@tlwu2013 Are we trying to handle the case where the CSV is deleted, but the Subscription still exists?